avocado-setup.py

```diff
@@ -377,7 +378,12 @@ def run_test(testsuite, avocado_bin):
     logger.info('')
     result_link = testsuite.jobdir()
     if result_link:
-        result_link += "/job.log"
+        result_json = result_link + "/results.json"
+        result_link += "/job.log\n"
+        with open(result_json, encoding = "utf-8") as fp:
+            result_state = json.load(fp)
+        for state in ['total', 'pass', 'cancel', 'errors', 'failures', 'skip']:
+            result_link += "|  %s %s  |" % (state, str(result_state[state]))
```

Comment:
and better to add a try: except here inorder to handle a missing key in different avocado version if any...

Reply:
Sure